Celeste.Mod.mm/Mod/Everest/Everest.cs

```diff
@@ -463,7 +463,7 @@ public static partial class Everest {
             }).Register();
 
             LuaLoader.Initialize();
-
+            TypeHelper.entityDataNameToType = new();
```

Comment:
This should just be done inside of BakeVanillaEntityData (or maybe `TypeHelper.Load()` or something simillar)

Reply:
Actually, this line completely cancels out `BakeVanillaEntityData`, making it impossible to use TypeHelper on vanilla entities

Reply:
that was meant to be the Type.FullName -> Type cache